test/fixtures/car.mpd

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<MPD xmlns="urn:mpeg:DASH:schema:MPD:2011" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xlink="http://www.w3.org/1999/xlink" xsi:schemaLocation="urn:mpeg:DASH:schema:MPD:2011 DASH-MPD.xsd" minBufferTime="PT2S" type="dynamic" profiles="urn:mpeg:dash:profile:isoff-live:2011" availabilityStartTime="2014-09-19T00:54:13" minimumUpdatePeriod="PT5S" timeShiftBufferDepth="PT1800S">
+<MPD xmlns="urn:mpeg:DASH:schema:MPD:2011" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xlink="http://www.w3.org/1999/xlink" xsi:schemaLocation="urn:mpeg:DASH:schema:MPD:2011 DASH-MPD.xsd" minBufferTime="PT2S" type="static" profiles="urn:mpeg:dash:profile:isoff-live:2011" availabilityStartTime="2014-09-19T00:54:13" minimumUpdatePeriod="PT5S" timeShiftBufferDepth="PT1800S">
```

Comment:
`dynamic` in this code triggers live-style playback. Given this is pointing to a VOD video, I think that's probably an unsafe simplification but I'm not familiar enough with MPD to know for sure.